CONTRIBUTING.md

```diff
@@ -0,0 +1,46 @@
+# Contributing to the W3C Process Document
+Thank-you for offering to contribute to the [W3C Process Document](https://www.w3.org/2017/Process-20170301/). Here we provide some details on how you can contribute. 
+
+Changes to the process document tend to fall into a few classes, and how the W3C handles these varies. Simple editorial changes may require little review, but more substantial changes will require a more substantial review and discussions by the W3C community. Please read the below, and get involved as explained. 
+
+If you have any questions, please [raise an issue](/w3c/w3process/issues).
+
+## Simple Editorial Changes
+For simple editorial changes, please feel free to raise a `pull request` or [raise an issue](/w3c/w3process/issues) on this repository. A simple editorial change can be things like spelling and grammar mistakes, broken links, and so on.
+
+Some changes may appear simple but may actually be more complicated than they appear. Don't worry - please continue to send your `pull request` or [raise an issue](/w3c/w3process/issues) and we can discuss. 
+
+## The W3C Process Community Group
+The W3C run a [Process Community Group](https://www.w3.org/community/w3process/) which is an open group created to manage discussions on developing the [W3C Process Document](https://www.w3.org/2017/Process-20170301/). Some suggestions for changes to the [W3C Process Document](https://www.w3.org/2017/Process-20170301/) will need to be made within the [Process Community Group](https://www.w3.org/community/w3process/), please see information below as to whether you need to take your suggestions to the group or can just raise a `pull request` or `issue`.
+
+## Changes that Affect the Way the W3C Operates but Do Not Change the Policy
+For changes that affect the way that the W3C operates but do not change policy please do the following:
+
+* [Raise an issue](/w3c/w3process/issues) on this repo
+* Make sure the [Process Community Group](https://www.w3.org/community/w3process/) adds the issue to it's agenda. This will allow you to explain why you think this is an improvement to the operation of the W3C (it's probably not helpful to provide a Pull Request until you have consensus on the approach).
```

Comment:
s/it's/its

Reply:
Also, it may well be helpful to produce a Pull Request, since that lets people see exact wording. And a lot of times, the **exact** wording is where the discussion is focused.